### ISSUE TITLE
[Bug] Talkback loses focus when pressing tabs button

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#8588]
## Steps to reproduce

1. Launch Fenix with TalkBack on
1. Visit any site
1. Activate tabs button in toolbar

### Expected behavior

View switches to home screen with no talkback cursor

### Actual behavior

TalkBack cursor should be on top search bar

**Comment by asadotzler** [comment]
Shouldn't the talkback cursor be on the last selected tab in the tabs list in the home screen? That's what's getting visibly highlighted when you activate the tabs button.

**Comment by asadotzler** [comment]
This is an accessibility P2. The feature is available and works but is difficult to use because of inconsiderate focus behavior. This kind of bug prevents Fenix from being a delightful experience for users with disabilities.

**Comment by eeejay** [comment]
The visible highlight is the selected state of that tab, not focus. If the focus lands anywhere in that screen it would be an improvement. Either on the "Open tabs" header or the selected tab.

**Comment by mcarare** [comment]
Closing this, as the issue will be tracked and handled in https://github.com/mozilla-mobile/fenix/issues/9640.
